bundles/org.openhab.binding.openwebnet/src/main/java/org/openhab/binding/openwebnet/internal/handler/OpenWebNetAlarmHandler.java

```diff
@@ -229,27 +233,29 @@ private void updateZoneState(WhatAlarm w) {
     }
 
     private void updateZoneAlarm(WhatAlarm w) {
+        LocalDateTime now = LocalDateTime.now(ZoneId.systemDefault());
+        DateTimeFormatter format = DateTimeFormatter.ofPattern("MM-dd-yyyy HH:mm:ss");
         switch (w) {
             case ZONE_ALARM_INTRUSION:
-                updateState(CHANNEL_ALARM_ZONE_ALARM, new StringType(ALARM_INTRUSION));
+                updateState(CHANNEL_ALARM_ZONE_ALARM, new StringType(ALARM_INTRUSION + " " + now.format(format)));
                 break;
             case ZONE_ALARM_TAMPERING:
-                updateState(CHANNEL_ALARM_ZONE_ALARM, new StringType(ALARM_TAMPERING));
+                updateState(CHANNEL_ALARM_ZONE_ALARM, new StringType(ALARM_TAMPERING + " " + now.format(format)));
                 break;
             case ZONE_ALARM_ANTI_PANIC:
-                updateState(CHANNEL_ALARM_ZONE_ALARM, new StringType(ALARM_ANTI_PANIC));
+                updateState(CHANNEL_ALARM_ZONE_ALARM, new StringType(ALARM_ANTI_PANIC + " " + now.format(format)));
                 break;
             case ZONE_ALARM_SILENT:
-                updateState(CHANNEL_ALARM_ZONE_ALARM, new StringType(ALARM_SILENT));
+                updateState(CHANNEL_ALARM_ZONE_ALARM, new StringType(ALARM_SILENT + " " + now.format(format)));
                 break;
             case ZONE_ALARM_TECHNICAL:
-                updateState(CHANNEL_ALARM_ZONE_ALARM, new StringType(ALARM_TECHNICAL));
+                updateState(CHANNEL_ALARM_ZONE_ALARM, new StringType(ALARM_TECHNICAL + " " + now.format(format)));
                 break;
             case ZONE_ALARM_TECHNICAL_RESET:
-                updateState(CHANNEL_ALARM_ZONE_ALARM, new StringType(ALARM_TECHNICAL_RESET));
+                updateState(CHANNEL_ALARM_ZONE_ALARM, new StringType(ALARM_TECHNICAL_RESET + " " + now.format(format)));
                 break;
             default:
-                updateState(CHANNEL_ALARM_ZONE_ALARM, new StringType(ALARM_NONE));
+                updateState(CHANNEL_ALARM_ZONE_ALARM, new StringType(ALARM_NONE + " " + now.format(format)));
```

Comment:
see previous comment

Reply:
I see, ok  I will keep working from the base you suggested. Do not consider this PR!